plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/HuaweiCloudMetricsService.java

```diff
@@ -88,7 +90,15 @@ public List<Metric> getMetricsByResource(ResourceMetricRequest resourceMetricReq
                     huaweiCloudToXpanseDataModelConverter.buildShowMetricDataRequest(
                             resourceMetricRequest, metricInfoList);
             ShowMetricDataResponse showMetricDataResponse =
-                    client.showMetricData(showMetricDataRequest);
+                    client.showMetricDataInvoker(showMetricDataRequest)
+                            .retryTimes(5)
```

Comment:
can we move the number of retries to a static constant variable please?

Reply:
ok,I made this change.